pkg/store/verifiable/store_test.go

```diff
@@ -47,14 +47,6 @@ var udCredential = `
 
   "issuanceDate": "2010-01-01T19:23:24Z",
 
-  "proof": {
```

Comment:
are these proofs being removed from a test because they are no longer valid?

Reply:
The test was failing... As we now check embedded proofs when unmarshalling `Credential` and thus we need to pass a way of how to get the public key for signature verification. Also, the removed proof was of "RsaSignature2018" type what is currently not supported in `signature` package (and thus in `verifiable` package as well).